packages/viz/src/utils/print/data.js

```diff
@@ -189,7 +189,7 @@ export function selectDailyViewData(medicalData, startDate, endDate) {
     bgRange: processBgRange(dataByDate),
     bolusRange: processBolusRange(dataByDate),
     dateRange: [startDate.format('YYYY-MM-DD'), endDate.format('YYYY-MM-DD')],
-    latestPumpUpload: getLatestPumpUpload(medicalData.medicalData.uploads),
+    pumpSettings: medicalData.medicalData.pumpSettings[0],
```

Comment:
Same question

Reply:
Given how `medicalData.medicalData` is initialized in `medical-data.service.ts` (line 81), I would say there is no risk of a null pointer exception: the `pumpSettings` property exists and is at worst equal to an empty array